tools/src/main/scala/org/apache/spark/tools/GenerateMIMAIgnore.scala

```diff
@@ -160,35 +167,13 @@ object GenerateMIMAIgnore {
    * and subpackages both from directories and jars present on the classpath.
    */
   private def getClasses(packageName: String): Set[String] = {
-    val path = packageName.replace('.', '/')
```

Comment:
These changes are just part of switching us to use ClassUtil for finding the Spark classes.